docs/how-to-debug-notifications.md

```diff
@@ -0,0 +1,77 @@
+# How to debug notifications
+
+## Get Firebase project id
+
+Two Firebase projects have been created, one for development environnement (_cozy-flagship-app-push-dev_) and one for production environment (_cozy-flagship-app-push-prod_).
+
+```
+import { utils } from   '@react-native-firebase/app'
+
+console.log(utils()._app._options.projectId) // cozy-flagship-app-push-dev or cozy-flagship-app-push-prod
+```
+
+## Get FCM token
```

Comment:
nit: Same here, why should I get the FCM token? (I read the next section so I have an answer but it would be great to have the answer directly in this section)